src/format.py

```diff
@@ -51,36 +53,35 @@ def update_context(self, tag):
             elif tag.type == TagType.TEXTSTYLE:
                 self.TS_tag(tag)
             elif tag.type == TagType.POP:
-                warnings.warn("Got POP tag in update_context(). pop_tag() should be called explicitly", SyntaxWarning)
+                warnings.warn(
```

Comment:
No idea what PEP8 says about it, but personally I prefer putting the closing paren on the next line, at the same indentation level as the open paren. Down for whatever though

Reply:
yeah, but the formatter does what it wants. we'll config better in a follow-up